scrapers/stackoverflow.js

```diff
@@ -0,0 +1,61 @@
+var unirest = require("unirest");
+var question = "";
+for (i = 2; i < process.argv.length; i++) {
+    question += process.argv[i] + " ";
```

Comment:
Please use a single parameter to take a query parameter. By convention different parameters are separated by space, in case a parameter is more than a word, users provide that parameter using double quotes e.g 

`node stackoverflow.js "Difference between remainder and modulus in number theory?"`